### DISCUSSION TITLE
10.1.0 -> /iphone/cli/commands/_buid.js Error

### DISCUSSION THREAD

**Discussion by mbender74** [appcelerator/titanium_mobile#12600] [Q&A] [answered]
ReferenceError: Cannot access 'flattenedDefaultIconDest' before initialization
    at iOSBuilder.createAppIconSetAndiTunesArtwork (/Users/marcbender/Library/Application Support/Titanium/mobilesdk/osx/10.1.0/iphone/cli/commands/_build.js:5782:64)

**Comment by Topener** [comment]
Hi, could you provide more information?

**Comment by ewanharris** [comment, marked as answer]
I filed https://jira.appcelerator.org/browse/TIMOB-28401 for this issue and will have a PR up shortly

Edit: PR is https://github.com/appcelerator/titanium_mobile/pull/12625

**Comment by mbender74** [comment]
Great ;-)
